dkg-gadget/src/async_protocols/mod.rs

```diff
@@ -274,12 +274,12 @@ impl OfflinePartyId {
 
 #[derive(Debug, Copy, Clone)]
 pub enum KeygenRound {
-	/// Keygen round is active
-	ACTIVE,
-	/// Keygen round is queued
-	QUEUED,
-	/// UNKNOWN
-	UNKNOWN,
+	/// Executed at the beginning of session 0
+	Genesis,
+	/// Executed at the end of session 0
+	GenesisNext,
```

Comment:
Why is this needed? We had Unknown state previously for error cases afaik, CC @shekohex